.github/workflows/build.yaml

```diff
@@ -260,6 +260,21 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh 'pip3 install ./out/controller/python/chip-0.0-cp37-abi3-linux_x86_64.whl'
                   scripts/run_in_build_env.sh '(cd src/controller/python/test/unit_tests/ && python3 -m unittest -v)'
+            - name: Build all clusters app for test
+              timeout-minutes: 50
+              run: |
+                  scripts/examples/gn_build_example.sh examples/all-clusters-app/linux out/all_clusters_debug chip_enable_wifi=false
+            - name: Run Python REPL API Tests
+              timeout-minutes: 5
+              run: |
+                  scripts/run_in_build_env.sh '
+                      rm -rf /tmp/chip*
+                      ./out/all_clusters_debug/chip-all-clusters-app &
```

Comment:
Also, I'd like to have a single convenient script that users can run that automatically launches both the all-clusters-app AND the mobile-test suite, as well as clearing out the contents of `/tmp/chip*`.

That script should automatically permit forwarding of arguments to the `mobile-device-test.py`.

Reply:
Added a script